modules/ROOT/pages/tutorial-autologin.adoc

```diff
@@ -142,6 +142,8 @@ The `virt-install` command will start an instance named `fcos` from the `fedora-
 
 We use the `backing_store` option to `virt-install --disk` to quickly create a new disk image and avoid writing to the original image we have downloaded. This new disk image can be easily thrown away.
 
+NOTE: Depending on your version of `virt-install`, you may not be able to use `--os-variant=fedora32` and will get an error. In this case, you should pick an older Fedora variant (`--os-variant=fedora31` or `--os-variant=fedora30` for example). You can find the variants that are supported by you current version of `virt-install` with `osinfo-query os | grep "fedora[2-3][0-9]"`.
```

Comment:
Should we add this blurb to https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-libvirt/ too?

Reply:
Good idea!